### ISSUE TITLE
Renaming / deleting common fields should be hidden

### ISSUE THREAD

**Issue by KenZync** [authpass/authpass#164]
My thought is renaming common fields is should not be possible may be hide it also delete button shouldn't exist at all except for custom fields

**Comment by hpoul** [comment]
I don't get it. If you rename a common field, it just turns into a custom field? :-)
Users can name the fields whatever they like, the "common" fields are just a convention. I see no good reason to prevent this.. and it's not like it takes up too much UI space, it's hidden in the context menu anyway? what are the benefits?

**Comment by KenZync** [comment]
The rename I don't argue with you but delete button should not appear in common field as it doesn't work anyway

**Comment by hpoul** [comment]
I agree, it doesn't work -- we should fix deleting of common fields :-P rather than making it impossible.
